db/migrate/20220520205001_create_organizations.rb

```diff
@@ -0,0 +1,14 @@
+class CreateOrganizations < ActiveRecord::Migration[6.1]
+  def change
+    create_table :organizations do |t|
+      t.string :name
+      t.string :address
+      t.int :phone
+      t.string :email
+      t.text :welcomeText
```

Comment:
La convención en Rails con respecto a los nombres compuestos es usar `snake case` en lugar de `camel case`:
```suggestion
      t.text :welcome_text, null: false
```

Reply:
Hecho